_posts/2016-04-09-linux-mint-razer-blade.markdown

```diff
@@ -0,0 +1,97 @@
+---
+layout: post
+title: Running Linux Mint on a Razer Blade
+tags: linux mint razer-blade setup
+username: likethemammal
+---
+
+![Linux Mint logo](https://upload.wikimedia.org/wikipedia/commons/5/5c/Linux_Mint_Official_Logo.svg)
+
+**tl;dr** The <a  href="http://www.amazon.com/gp/product/B01AGINE7Q/ref=as_li_tl?ie=UTF8&camp=1789&creative=9325&creativeASIN=B01AGINE7Q&linkCode=as2&tag=likethemammal-20&linkId=VLVDTKUEZSESNTID">Razer Blade 2015</a><img src="http://ir-na.amazon-adsystem.com/e/ir?t=likethemammal-20&l=as2&o=1&a=B01AGINE7Q" width="1" height="1" border="0" alt="" style="border:none !important; margin:0px !important;" /> is a great portable gaming workstation. Linux Mint runs beautifully with some tweaking, that I've boiled down after months of usage. The only problem still unresolved is an issue with the internal webcam.
+
+###Prologue
+I've been looking for a portable gaming PC for a while. Last year, [Lifehacker's review of the 2015 Razer Blade Full HD](http://gizmodo.com/razer-blade-2015-review-finally-living-the-thin-gaming-1689073179) convinced me that was the machine I've been waiting for.
+
+A few months back Razer finally put that model up <a  href="http://www.amazon.com/gp/product/B01AGINE7Q/ref=as_li_tl?ie=UTF8&camp=1789&creative=9325&creativeASIN=B01AGINE7Q&linkCode=as2&tag=likethemammal-20&linkId=VLVDTKUEZSESNTID">on Amazon</a><img src="http://ir-na.amazon-adsystem.com/e/ir?t=likethemammal-20&l=as2&o=1&a=B01AGINE7Q" width="1" height="1" border="0" alt="" style="border:none !important; margin:0px !important;" />, at a discount. With the power of a credit card, I finally had the mini gaming workstation of my dreams in hand!
+
+I've done 3 or 4 full builds of Linux before, all with Mint. One on a Chromebook, but mostly on machines with slightly dated hardware. This build has by far been the hardest, and I learned the hard way that Linux isn't as stable on "new" hardware.
+
+###Chapter 1: Pre-install
+**Expand SSD storage (optional)**
+
+The SSD in the Full HD edition is about 250 GB. If you're dual-booting, like I did, this doesn't leave much space to split. If you're going to replace the ssd with a larger one, now's the time to do it. Heres a video on how:
```

Comment:
Don't wanna be specific about the number of gigs cus the os's and rando partitions take up space here or there. Its probably closer to 100-110 each.

Also, the partitions don't have to be split down the middle. Just wanna suggest that 250 is much to split between two os's